### ISSUE TITLE
documentation is to sparse

### ISSUE THREAD

**Issue by rofl0r** [Shizmob/vm-utils#2]
could you please elaborate on the "like online.net" setup and on the vnc/tmux stuff you use ? maybe some screenshots could help, too.

**Comment by shizmob** [comment]
Yeah, I wrote it at like 5AM before pushing the code. I'll expand it a bit more over the weekend.

The "like Online.net" part is where Online.net hands you IPv4s that are MAC-locked and VMs can access their relevant packets directly over the host interface: this could also be solved using a normal bridge interface, but it is my understanding that macvtap is more secure against VMs sniffing on eachother's traffic. In contrast, their IPv6 setup delegates an entire /48 (or /56, or however you choose to subnet it) to your host machine using DHCPv6: you have to route the IPv6 traffic manually, which can't be done over macvtap as the host can not reach the guests in those interfaces. For that, we add a second regular tap interface and some routing rules in the kernel, which preserved VM network isolation.

The tmux stuff basically comes down to it creating a QEMU management console at the `vm-<name>` tmux session. You can press CTRL-A C in that console to get access to the QEMU command line. For VNC things, I have it personally always bind to localhost and use SSH port forwarding for security, as VNC password security is rather bad.

These are all things I want to add to the README at the very least, is there anything missing or unclear in this for you?
